generator/src/main/java/com/scottlogic/deg/generator/utils/JavaUtilRandomNumberGenerator.java

```diff
@@ -48,4 +50,12 @@ public int nextInt(int lowerInclusive, int upperExclusive) {
     public double nextDouble(double lowerInclusive, double upperExclusive) {
         return random.nextDouble() * (upperExclusive - lowerInclusive) + lowerInclusive;
     }
+
+    @Override
+    public BigDecimal nextBigDecimal(double lowerInclusive, double upperExclusive, int scale) {
```

Comment:
Shouldn't these parameters also be `BigDecimal`s?

Reply:
yep, done